mibidata/mibi_image.py

```diff
@@ -342,6 +333,34 @@ def parse_aperture(value):
                 )
         return aperture
 
+
+    @staticmethod
+    def match_fov_folder(value, field, value_name,
```

Comment:
Nit pick comment, but shouldn't this be match_field or something similar? If we're passing in a generic parameter names then should the function name also be generic? I also prefer validate to match in cases like that, but Python might disagree.